examples/node-mssql-programmatic-use/connections.sync-db.json.example

```diff
@@ -2,12 +2,14 @@
   "connections": [
     {
       "id": "db1",
-      "host": "localhost",
-      "port": 1433,
-      "user": "db1user",
-      "database": "db1",
-      "password": "password",
-      "client": "mssql"
+      "client": "mssql",
+      "connection": {
+        "host": "localhost",
+        "port": 1433,
+        "user": "sa",
+        "database": "tempdb",
+        "password": "Password@123"
```

Comment:
@samirsilwal You did the same thing here. Please set is as `Secret@123`. Copy pasting this example file should work out of the box. 🙏